web/app/themes/huskies-theme/functions/bbpress/forum.php

```diff
@@ -23,19 +23,8 @@ function subscribe_link($args = array()) {
   function is_unread() {
     if (isset($this->unread)) return $this->unread;
 
-    $query = array(
-      'post_type' => bbp_get_topic_post_type(),
-      'post_parent' => $this->id,
-      'meta_key' => 'bbppu_read_by',
-      'meta_value' => 'i:'.bbp_get_current_user_id(),
-      'meta_compare' => 'NOT LIKE',
-      'orderby' => 'title',
-      'order' => 'DESC',
-      'nopaging' => true
-    );
-
-    $topics = new WP_Query($query);
-    $this->unread = count($topics->posts) > 0;
+    $topics = new WP_Query(get_unread_topic_query_args($this->id));
```

Comment:
Can you use the function `get_unread_counts` (https://github.com/Ultimate-Huskies/website/pull/4/files#diff-6ccbaf6ce36cb0e19eba92b9ee80fc7aR68) directly?

Reply:
Unfortunately not. `get_unread_counts` retrieves **all** unread topics. Here, I only want to get all unread topics of a **specified forum**.